plugins/MauticCrmBundle/Integration/VtigerIntegration.php

```diff
@@ -173,7 +173,7 @@ public function authCallback($settings = [], $parameters = [])
      */
     public function getAvailableLeadFields($settings = [])
     {
-        if ($fields = parent::getAvailableLeadFields()) {
+        if ($fields = parent::getAvailableLeadFields($settings)) {
```

Comment:
The question I have about this change is why is `getAvailableLeadFields` method even overwritten? Can we just delete it from this child class? Or is it important that if there are no `$fields`, it will return `null` instead of `[]`?

Reply:
As I understand, It's just cache.  I saw some issue after Mautic update, then parent::getAvailableLeadFields returns everytime just leads object, not company object. This minor PR resolve it.
Maybe we could check empty() on return empty array.